app/models/order_number/definition.rb

```diff
@@ -40,6 +40,10 @@ def id
       @id ||= quota_definition_sid
     end
 
+    def all_goods_nomenclatures
+      measures.map(&:goods_nomenclature).uniq(&:goods_nomenclature_item_id).sort_by(&:goods_nomenclature_item_id)
```

Comment:
this needs to handle measures without goods nomenclatures (ie `.compact` after `.map`)